frontend/src/components/RequestFormInputs.vue

```diff
@@ -228,8 +224,18 @@ const executeFaucetRequest = async () => {
 };
 const { active: faucetRequestActive, run: runFaucetRequest } =
   createAsyncProcess(executeFaucetRequest);
+
+const gasFeesTooltipText = `The fee amount is composed of three parts:
+  - the gas reimbursement fee
+  - the liquidity provider fee
+  - the Beamer service fee
+The gas reimbursement fee and the liquidity provider fee are paid out to the liquidity provider servicing the request, while the Beamer service fee stays with the contract and supports the Beamer platform development.
```

Comment:
```suggestion
The gas reimbursement fee and the liquidity provider fee are paid out to the liquidity provider servicing the request, while the Beamer service fee stays with the contract and supports the Beamer platform development.

```

Reply:
What is the suggestion here exactly, couldn't figure it out?

Reply:
To separate the middle paragraph with an empty line, above and below, in order to improve readability. Just my personal preference.